openwisp_monitoring/check/base/models.py

```diff
@@ -65,7 +65,7 @@ def check_class(self):
         """
         returns check class
         """
-        return import_string(self.check)
+        return import_string(self.check_type)
```

Comment:
because of this, I think we should call this `check_class` instead of `check_type`, because the content of this field is a python path leading to a python class, it makes the code more explicit and hence more readable.

Reply:
There is already a property of Check model name check_class. 

https://github.com/openwisp/openwisp-monitoring/blob/5b8169707783e0078e6541044635a48826a918f9/openwisp_monitoring/check/base/models.py#L63-L68